src/index.html

```diff
@@ -1,13 +1,50 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div data-qa="card" class="card">
+      <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="imac A419"
+      >
+      <p class="card__name">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </p>
+      <p class="card__code">
+        Код товара: 195434
+      </p>
+      <div class="card__stars">
+        <div class="stars stars--4">
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+        </div>
+        <p class="card__review">
+          Отзывов: 5
+        </p>
+      </div>
+      <div class="card__price">
+        <p class="price">Цена:</p>
+        <p class="uah">69 999 грн</p>
+      </div>
```

Comment:
```suggestion
      <div class="card__price">
        <p class="card__price-label">Цена:</p>
        <p class="card__price-value">69 999 грн</p>
      </div>
```
There's no need to make these elements separate BEM-blocks